src/vulkan/graphics_pipeline.cc

```diff
@@ -767,42 +772,41 @@ Result GraphicsPipeline::Clear() {
   colour_clear.color = {
       {clear_color_r_, clear_color_g_, clear_color_b_, clear_color_a_}};
 
-  Result r = ClearBuffer(colour_clear, VK_IMAGE_ASPECT_COLOR_BIT);
```

Comment:
Why was the method removed and this merged into one big function?

Reply:
Because it contained code that should've been done only once before copying color and depth/stencil buffer. Even the original implementation copied color buffers twice if depth buffer was present. Basically there was nothing left in the function when fixing this.